src/qpmodel.jl

```diff
@@ -50,32 +49,14 @@ end
 
 linobj(qp::AbstractQuadraticModel, args...) = qp.data.c
 
-function objgrad(qp :: AbstractQuadraticModel, x :: AbstractVector)
-    g = Vector{eltype(x)}(length(x))
-    objgrad!(qp, x, g)
-end
-
-function objgrad!(qp :: AbstractQuadraticModel, x :: AbstractVector, g :: AbstractVector)
-    v = qp.data.opH * x
-    @. g = qp.data.c + v
-    f = qp.data.c0 + dot(qp.data.c, x) + 0.5 * dot(v, x)
-    qp.counters.neval_hprod += 1
-    (f, g)
-end
```

Comment:
I don't think you should have removed `objgrad!` because it's more efficient in this case than the default (which calls `obj` and `grad` successively).